git2rss

```diff
@@ -1,5 +1,15 @@
 #!/usr/bin/env perl -w
-#
+# 
+# Notes::
+# If you see the error `/usr/bin/env: ‘perl -w’: No such file or directory
```

Comment:
The correct fix is to remove the `-w` from the shebang line.

The `use warnings ;` on line 42 does the same thing as the `-w` option here.